### PR TITLE
Add service management params

### PR DESCRIPTION
**Pull request by jcpunk**
There are a few ways to check if mcelog is supported by the hardware.  I may have to do some research before writing up something to make this smarter.

For now, on my site, I can just use these params to override hosts where it doesn't work.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -7,13 +7,19 @@
...
+# @param service_enable
+#   Should the service start on boot?
 # @param config_file_content
 #   Literal string to write to config_file_path.
 #
 class mcelog (
   String $package_name,
   Stdlib::Absolutepath $config_file_path,
   String $service_name,
+  Enum["running", true, "stopped", false] $service_ensure,
```
Comment (jhoblitt) [inline manifests/init.pp:21]: How about using `Stdlib::Ensure::Service` instead of a manual enum?

Comment (jcpunk) [reply manifests/init.pp]: good call, I forgot that existed.

**Review by jhoblitt — APPROVED** [review]

**Comment by jhoblitt** [comment]
Try `bundle exec rake rubocop:auto_correct`

**Comment by jhoblitt** [comment]
OK, looks like I need to change the github workflows to run on PRs.

**Comment by jhoblitt** [comment]
#17 fixes the github action config.

**Comment by jcpunk** [comment]
Looks like the test failures are related to github timeouts?

**Comment by jhoblitt** [comment]
Ya. I'm restarting the failures....  For some reason it is only letting me kick one at a time.

**Comment by jhoblitt** [comment]
released as `2.1.0`.

**Pull request merged**